mender-update/daemon/states.cpp

```diff
@@ -542,6 +542,14 @@ void EndOfDeploymentState::OnEnter(Context &ctx, sm::EventPoster<StateEvent> &po
 	poster.PostEvent(StateEvent::Success);
 }
 
+ExitState::ExitState(events::EventLoop &event_loop) :
+	event_loop_(event_loop) {
+}
+
+void ExitState::OnEnter(Context &ctx, sm::EventPoster<StateEvent> &poster) {
+	event_loop_.Stop();
+}
```

Comment:
Both of these can easily be in the `.hpp` file.

Reply:
I know, but due to the sheer number of states, I prefer to keep all the code in the `.cpp` file so that it's easier to navigate.